File: 800/Drinks/drinks.py
# https://codeforces.com/problemset/problem/200/B

a = []      # empty array list
sum = 0     # stores the sum of array elements in it

# element number in array
n = int(input("Number of elements in array:"))
for i in range(0,n):
   l = int(input())
   a.append(l)
print(a)
for i in range(0, n):
  sum = sum + a[i]

print("The sum is: ", sum)
temp = n * 100
val = sum % temp
val1 = val / n
print("Percentage: ", val1)      # calculate the percentage

File: 800/Drinks/drinks.cpp
// https://codeforces.com/problemset/problem/200/B

#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter the drinks: ";
    cin >> n;
    int arr[100];
    int sum = 0;
    for(int i = 0; i < n; i++) {
        cout << "Enter the volume: ";
        cin >> arr[i];
    }
    cout << "The array is: ";
    for(int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    cout << "\n";
    for(int i = 0; i < n; i++) {
        sum = sum + arr[i];
    }
    int temp_n = n * 100;
    float ans = sum % temp_n;
    float ans1 = ans / n;
    cout << "Percentage: " << ans1;
    return 0;
}
